post-content.php

```diff
@@ -145,11 +145,11 @@
 <p><?php _e( 'Any block can opt into these alignments. The embed block has them also, and is responsive out of the box:', 'gutenberg' ); ?></p>
 <!-- /wp:paragraph -->
 
-<!-- wp:core-embed/vimeo {"url":"https://vimeo.com/22439234","type":"video","providerNameSlug":"vimeo","align":"wide","className":"wp-has-aspect-ratio wp-embed-aspect-16-9"} -->
-<figure class="wp-block-embed-vimeo alignwide wp-block-embed is-type-video is-provider-vimeo wp-has-aspect-ratio wp-embed-aspect-16-9"><div class="wp-block-embed__wrapper">
```

Comment:
Hm, are we removing these classes (`wp-block-embed-${type}`) from the block output? We should keep them, they are important for compat.

Reply:
I'll change it back - is this needed for possible theme style overrides?

Reply:
Yeah, that's the idea.